src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,235 @@
+    <cluster code="0x0099"/>
+    <item name="EvStopped"    value="0x00"  optional="false"/>
+    <item name="EvseStopped"  value="0x01"  optional="false"/>
+    <item name="Other"        value="0x03"  optional="false"/>
+  </enum>
+
+  <struct name="ChargingTargetStruct">
+    <cluster code="0x0099"/>
+    <item name="TargetTime"   type="INT16U"  optional="false"/>
+    <item name="TargetSoC"    type="INT8U"   optional="true"/>
+    <item name="AddedEnergy"  type="INT32U"  optional="true"/>
+  </struct>
+
+  <cluster>
+    <name>Energy EVSE</name>
+    <domain>Energy Management</domain>
+    <description>This cluster provides an interface for the management of Electric Vehicle Supply Equipment (EVSE).</description>
+    <code>0x0099</code>
+    <define>ENERGY_EVSE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+
+    <attribute side="server" code="0x0000" define="STATE"                       type="EvseStateEnum"              writable="false"                  optional="false" isNullable="true">State</attribute>
+    <attribute side="server" code="0x0001" define="SUPPLY_STATE"                type="SupplyStateEnum"            writable="false"                  optional="false" isNullable="false">SupplyState</attribute>
+    <attribute side="server" code="0x0002" define="FAULT"                       type="FaultStateEnum"             writable="false"                  optional="false" isNullable="false">Fault</attribute>
+    <attribute side="server" code="0x0003" define="ENABLE_CHARGE_TIME"          type="elapsed_s"                  writable="false"                  optional="false" isNullable="true">EnableChargeTime</attribute>
+    <attribute side="server" code="0x0004" define="ENABLE_DISCHARGE_TIME"       type="elapsed_s"                  writable="false"                  optional="false" isNullable="true">EnableDischargeTime</attribute>
+    <attribute side="server" code="0x0005" define="CIRCUIT_CAPACITY"            type="INT32U"  writable="false" default="0x0"    optional="false" isNullable="false">CircuitCapacity</attribute>
+    <attribute side="server" code="0x0006" define="MINIMUM_CHARGE_CURRENT"      type="INT32U"  writable="false" default="0x1770" optional="false" isNullable="false">MinimumChargeCurrent</attribute>
+    <attribute side="server" code="0x0007" define="MAXIMUM_CHARGE_CURRENT"      type="INT32U"  writable="false" default="0x0"    optional="false" isNullable="false">MaximumChargeCurrent</attribute>
+    <attribute side="server" code="0x0008" define="MAXIMUM_DISCHARGE_CURRENT"   type="INT32U"  writable="false" default="0x0"    optional="false" isNullable="false">MaximumdDischargeCurrent</attribute>
+    <attribute side="server" code="0x0009" define="USER_MAXIMUM_CHARGE_CURRENT" type="INT32U"  writable="true"  default="0x0"    optional="true"  isNullable="false">UserMaximumChargeCurrent</attribute>
```

Comment:
Access for write here and below seem to be `manage`. Please add this to XML (generally name goes in description and then you add a `<access op="write" role="manage"/>` as well.

Reply:
Changed.